Import prebuilt package and common scripts from MediaPipeUnityPlugin.

henryouly opened the pull request:
Import prebuilt package and common scripts from MediaPipeUnityPlugin

Built from commit [ee27e24e963c5485a36f5a63fc7215e732f4d49f](https://github.com/homuler/MediaPipeUnityPlugin/commit/ee27e24e963c5485a36f5a63fc7215e732f4d49f)

Copied:
- `Packages/com.github.homuler.mediapipe` to `Assets/Packages/com.github.homuler.mediapipe`
- `Assets/Mediapipe/Samples/Common` to `Assets/Scripts/Mediapipe/Common`
- `Assets/Mediapipe/Samples/Scenes/Hand Tracking` to `Assets/Scripts/Mediapipe/Hand Tracking`. (Deleted prefab, `HandTrackingSolution` and `HandTrackingConfig`)
- `Assets/Mediapipe/Samples/Scenes/Pose Tracking` to `Assets/Scripts/Mediapipe/Pose Tracking`. (Deleted prefab, `PoseTrackingSolution` and `PoseTrackingConfig`)
- Added `Test.cs` to make sure compilation is successful.
- Files are copied as is.

// TODO: How to make a hello world unit test to cover this setup?

Import Unity Chan model from Unit Asset Store

- Removed unused assets like voice and documentation.
- Add the model to `Sample Scene` to test completeness.

wixette commented:
## Git LFS

Please consider using [Git LFS](https://git-lfs.github.com/) in this project so that large asset files can be stored separately and be managed easily. Here is a sample config file for Git LFS in a Unity project:

https://github.com/SeedV/SeedCalc/blob/main/.gitattributes

## 3rd-party lincense

The "Unity-chan! Model" is a third-party asset released under its own license. Considering the copyright compliance, I'd suggest to add a `README.md` file to the dir `Assets/Models/Unity-chan! Model`. The file could contain the following info:

> [Unity-chan](https://unity-chan.com/) is an original character for developers provided by Unity Technologies Japan. It's released under the [Unity-chan License](https://unity-chan.com/contents/license_en/).

MediaPipe is not the same case since its source code has already had its own copyright info.

## Coding style

`Assets/Scripts/Mediapipe/Test.cs`

1. Which coding style are we going to follow for our own C# code? SeedCalc is following the [Google C# Style](https://google.github.io/styleguide/csharp-style.html). I'd suggest we do the same thing here.

2. Can Test.cs be put into a sub dir, such as `/Assets/Scripts/Mediapipe/Examples`?

3. It'd be better not to name it as `Test` if it is not a real unittest code.

henryouly commented:
Applied changes.

- Added `.gitattributes` to declare metadata for LFS as suggested.
- Added `Assets/Models/Unity-chan! Model/README.md` as suggested.
- Removed `Assets/Scripts/Mediapipe/Test.cs`. This file is meant for temporary use. It's only used to test importing the Unity plugin and no compile error is caused by the file, and I planned to remove it in a follow-up CL. Simply skip checking in this file.
